src/bin/nydus-image/main.rs

```diff
@@ -758,7 +758,11 @@ fn main() -> Result<()> {
             }
         }
     } else if let Some(matches) = cmd.subcommand_matches("merge") {
-        Command::merge(matches, &build_info)
+        let result = Command::merge(matches, &build_info);
+        if let Err(MergeError::InconsistentFilesystem(_)) = result {
+            std::process::exit(2);
```

Comment:
Does this will print the error message to stderr before exit?

Reply:
it seems not,may be use `eprintln`?